edisgo/flex_opt/reinforce_measures.py

```diff
@@ -725,3 +733,434 @@ def _replace_by_parallel_standard_lines(lines):
+
+        Parameters
+        ----------
+        edisgo_obj: class:`~.EDisGo`
+        grid: `~.network.grids.LVGrid`
+        bus_lv: Identifier of lv bus
+        bus_mv: Identifier of mv bus
+
+        Returns
+        ----------
+        transformer_changes=    dict
+        """
+        if bus_lv not in edisgo_obj.topology.buses_df.index:
+            raise ValueError(
+                f"Specified bus {bus_lv} is not valid as it is not defined in "
+                "buses_df."
+            )
+        if bus_mv not in edisgo_obj.topology.buses_df.index:
+            raise ValueError(
+                f"Specified bus {bus_mv} is not valid as it is not defined in "
+                "buses_df."
+            )
+
+        try:
+            standard_transformer = edisgo_obj.topology.equipment_data[
+                "lv_transformers"
+            ].loc[
+                edisgo_obj.config["grid_expansion_standard_equipment"][
+                    "mv_lv_transformer"
+                ]
+            ]
+        except KeyError:
+            raise KeyError("Standard MV/LV transformer is not in the equipment list.")
+
+        transformer_changes = {"added": {}}
+
+        transformer_s = grid.transformers_df.iloc[0]
+        new_transformer_name = transformer_s.name.split("_")
+        grid_id_ind = new_transformer_name.index(str(grid.id))
+        new_transformer_name[grid_id_ind] = lv_grid_id_new
+
+        transformer_s.s_nom = standard_transformer.S_nom
+        transformer_s.type_info = None
+        transformer_s.r_pu = standard_transformer.r_pu
+        transformer_s.x_pu = standard_transformer.x_pu
+        transformer_s.name = "_".join([str(_) for _ in new_transformer_name])
+        transformer_s.bus0 = bus_mv
+        transformer_s.bus1 = bus_lv
+
+        new_transformer_df = transformer_s.to_frame().T
+
+        old_s_nom = grid.transformers_df.s_nom.sum()
+
+        max_iterations = 10
+        n = 0
+
+        while old_s_nom > new_transformer_df.s_nom.sum() and n < max_iterations:
+            n += 1
+
+            another_new_transformer = new_transformer_df.iloc[-1:, :]
+
+            old_name = another_new_transformer.index[0]
+
+            name = old_name.split("_")
+
+            try:
+                name[-1] = str(int(name[-1]) + 1)
+            except ValueError:
+                name.append("1")
+
+            name = "_".join(name)
+
+            another_new_transformer.rename(index={old_name: name}, inplace=True)
+
+            new_transformer_df = pd.concat(
+                [new_transformer_df, another_new_transformer]
+            )
+
+        edisgo_obj.topology.transformers_df = pd.concat(
+            [edisgo_obj.topology.transformers_df, new_transformer_df]
+        )
+        transformer_changes["added"][
+            f"LVGrid_{lv_grid_id_new}"
+        ] = new_transformer_df.index.tolist()
+
+        return transformer_changes
+
+    G = grid.graph
+
+    # main station
+    station_node = list(G.nodes)[0]
```

Comment:
Does the station bus always have to be the first node? I am not sure, as G is an undirected and unordered graph.

Reply:
`station_node` is no identified from the `transformers_df`:

https://github.com/openego/eDisGo/blob/48255d3f4ed21f750fb4b0ddb76791d1bd28c660/edisgo/flex_opt/reinforce_measures.py#L925